src/foam/nanos/notification/email/SupportConfig.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.email',
```

Comment:
please move it "foam.nanos.app"
```suggestion
  package: 'foam.nanos.app',
```